bin/hipfc

```diff
@@ -59,7 +59,7 @@ function usage(){
     -hipfort-compiler <bin>   $HIPFORT_COMPILER or gfortran
     -hipfort          <path>  $HIPFORT   or _HIPFORT_INSTALL_DIR_
     -cuda-dir         <path>  $CUDA_HOME or /usr/local/cuda
-    -rocm-dir         <path>  $ROCM_DIR  or /opt/rocm
```

Comment:
I would prefer if we change the argument to `-rocm-path` to be a little more consistent.
We should probably switch to `-cuda-path` or `-cuda-home` from `-cuda-dir`.